src/graph_compute/update_function.h

```diff
@@ -43,7 +43,7 @@ using namespace std;
     phys_t timeStep;
     phys_t restLength;
   #if TEST_CONVERGENCE
-    phys_t * averageSpeed;
+    phys_t * meanSquareNetForce;
```

Comment:
We collect mean-square values and then divide by the total contributions and take the square root at the end, in getConvergenceData(..).